src/axolotl/utils/models.py

```diff
@@ -115,9 +116,9 @@ def load_model(
         logging.info("patching with sdp attention")
         hijack_llama_sdp_attention()
 
-    if cfg.bf16:
+    if cfg.bf16 or cfg.bfloat16:
```

Comment:
Why is there bloat16?

Reply:
when setting bf16, that's a HF AMP trainer specific thing, whereas we can alternatively use bloat16 without enabling HF AMP